lib/memory/index.js

```diff
@@ -58,13 +58,13 @@ module.exports = function MemoryStore(data_set) {
 
   const update = R.curry((bucket, id, data) =>
 
-      getById(bucket, id)
+    getById(bucket, id)
 
     .tap(R.when(R.isNil, () => throwItemNotFound(bucket, id)))
 
     .then(() => { store[bucket][id] = data })
 
-    .return(id)
+    .return(1)
```

Comment:
that affected rows count is coming from pimp-my-sql. if I change it there, it's going to affect all repos using pimp-my-sql.